index.php

```diff
@@ -1,128 +1,128 @@
 <!doctype html>
 <html lang="en">
-  <head>
-  	<title>Contact Management System</title>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+<head>
+<title>Contact Management System</title>
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
 
-	<link href='https://fonts.googleapis.com/css?family=Roboto:400,100,300,700' rel='stylesheet' type='text/css'>
+<link href='https://fonts.googleapis.com/css?family=Roboto:400,100,300,700' rel='stylesheet' type='text/css'>
 
-	<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
-	
-	<link rel="stylesheet" href="css/style.css">
+<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
 
-	</head>
-	<body>
-	<section class="ftco-section">
-		<div class="container">
-			<div class="row justify-content-center">
-				<div class="col-md-6 text-center mb-5">
-					<h2 class="heading-section text-success">Contact Form</h2>
-				</div>
+<link rel="stylesheet" href="css/style.css">
+
+</head>
```

Comment:
Everything in the head is a child of <html>, hence it should be:
html
 ---- head

The reason is because children html tags are always indented. A child element should not begin on the same line as the parent element

Reply:
Same thing will apply to the children elements of the head tag. The meta tags, link tags etc must be indented.